src/main/java/com/playmonumenta/worlds/paper/WorldManagementPlugin.java

```diff
@@ -172,28 +153,52 @@ public static boolean allowInstanceAutocreation() {
 		return mAllowInstanceAutocreation;
 	}
 
-	public static int getPregeneratedInstances() {
-		return mPregeneratedInstances;
-	}
-
-	public static int getUnloadInactiveWorldAfterTicks() {
-		return mUnloadInactiveWorldAfterTicks;
-	}
+	public static @Nullable ShardInfo getShardInfo(Player player) {
+		// TODO: For now, just use the current shard name.
+		// Eventually need some sorcery to let a player select a different entry
+		String shardName;
+		try {
+			shardName = NetworkRelayAPI.getShardName();
```

Comment:
I think this needs to get the first one, rather than this name, since it won't work with our instancing (brown, brown-2, etc.)

Reply:
Done.